google-account-plugin/src/com/google/cloud/tools/intellij/stats/GoogleUsageTracker.java

```diff
@@ -41,14 +41,36 @@
 import java.io.IOException;
 import java.util.List;
 
-/**
- * Google Usage Tracker that reports to Cloud Tools Analytics backend.
- */
+/** Google Usage Tracker that reports to Cloud Tools Analytics backend. */
 public class GoogleUsageTracker implements UsageTracker, SendsEvents {
 
   private static final Logger logger = Logger.getInstance(GoogleUsageTracker.class);
 
   private static final String ANALYTICS_URL = "https://ssl.google-analytics.com/collect";
+  private static final String PROTOCOL_VERSION_KEY = "v";
+  private static final String UNIQUE_CLIENT_ID_KEY = "cid";
+  private static final String NON_INTERACTIVE_KEY = "ni";
+  private static final String PAGE_TYPE_KEY = "t";
```

Comment:
It's actually "hit type", for which possible values are "pageview", "event", "exception", etc:

https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#t